sweep/sweeper.go

```diff
@@ -70,16 +70,15 @@ type Params struct {
 }
 
 // pendingInput is created when an input reaches the main loop for the first
-// time. It tracks all relevant state that is needed for sweeping.
+// time. It wraps the input and tracks all relevant state that is needed for
+// sweeping.
 type pendingInput struct {
+	input.Input
```

Comment:
why this change exactly?

Reply:
Two reasons:
1. Less code
2. Allows us to use `pendingInput` as `txInput` in next commits.